app/src/main/res/values/strings.xml

```diff
@@ -1119,7 +1119,6 @@
     <string name="show_more">show more</string>
     <string name="server_address_content_description">server address: %1$s</string>
     <string name="member_only_allowed">join as a member to access this feature</string>
-    <string name="select_theme_mode">select theme mode</string>
     <string name="beta_resources_auto_download">Beta resources auto download</string>
     <string name="select_theme_mode">select theme mode</string>]
```

Comment:
Hi @Kuljeet1998  ] bracket present at the end of the select_theme_mode

Reply:
I don't see the bracket but it should not be there

Reply:
Okay I have removed the bracket - please check @Okuro3499